src/providers/Provider.ts

```diff
@@ -1,9 +1,10 @@
 import { Readable } from 'stream'
-export type Message = { [key: string]: string }
+export type MessageValue = string
+export type Message = { [key: string]: MessageValue }
```

Comment:
What is the purpose of this indirection here, isn't MessageValue just a string?

Reply:
Because I am using the typing of MessageValue in the GovUKPayPaymentEventMessage. It means typescript wont allow you to accidentally pass this instead of the string you mean to pass.

It's the same as Alex B's arguments about naming types even if they are simple, this way it's always a MessageValue, and not (for example) a message key

Reply:
Gotcha 👍